data/macos.yaml

```diff
@@ -600,6 +658,14 @@ urls:
 - 'http://forensicswiki.org/wiki/Mac_OS_X'
 - 'http://forensicswiki.org/wiki/Mac_OS_X_10.9_-_Artifacts_Location#Skype'
 ---
+name: MacOSSkypeMainDirectory
```

Comment:
Seeing this is a cross platform application I opt to move this to applications definitions file

Reply:
ok, I will remove it. Given that there were already Skype related artifacts I just added some new ones. I will start an im application specific yaml file with those in it.

Reply:
Fixed